tree/treeplayer/src/TTreeProcessorMT.cxx

```diff
@@ -43,17 +43,17 @@ struct EntryCluster {
 
 ////////////////////////////////////////////////////////////////////////////////
 /// Construct fChain, also adding friends if needed and injecting knowledge of offsets if available.
-void TTreeView::MakeChain(const std::string &treeName, const std::vector<std::string> &fileNames,
+void TTreeView::MakeChain(const std::vector<std::string> &treeNames, const std::vector<std::string> &fileNames,
```

Comment:
```suggestion
/// \param[in] treeNames Name of the tree for each file in `fileNames`.
/// \param[in] fileNames Files to be opened.
/// \param[in] friendInfo xxx No idea what this is.
/// \param[in] nEntries Number of entries to be processed.
/// \param[in] friendEntries xxx Why do we need that? 
void TTreeView::MakeChain(const std::vector<std::string> &treeNames, const std::vector<std::string> &fileNames,
```

Reply:
Maybe it's just an internal function, but certainly doesn't do any harm.

Reply:
Yes, TTreeView is an internal helper class. Thanks though, committed!